resources/sass/layouts/_main.scss

```diff
@@ -2,3 +2,55 @@
     font-family: $ff-open-sans;
     margin: 0 auto;
 }
+
+.ris-content {
+    $base-content: &;
+    position: relative;
+
+    &_six-eight-eight {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 688px;
+
+        @include breakpoint($bp-tablet-min) {
+            .ris-welcome {
+                &__content {
+                    flex-basis: 668px;
+                    padding-right: $indent-ml;
+                }
+
+                &-map__desktop {
+                    margin-left: 668px + $indent-ml;
+                    right: auto;
+                }
+            }
+
+            &#{$base-content}_has-widget {
+                max-width: 1024px;
```

Comment:
this one look like a media query breakpoint, @krausweb do you maybe have a variable for it?

Reply:
Ok, thanks, 
I will fix there and similar comments during my the task Detail page PR
https://github.com/nextsolx/ratsinformationssystem/pull/17